src/info.rs

```diff
@@ -362,6 +361,24 @@ impl Info {
         })
     }
 
+    async fn get_git_history(dir: &str, no_merges: bool) -> Vec<String> {
+        let mut args = vec!["-C", dir, "log"];
+        if no_merges {
+            args.push("--no-merges");
+        }
+
+        args.push("--pretty=format:%cr%x09%an");
```

Comment:
Just for slightly more readability (and keeps the character a literal instead of git having to parse it). The tab character doesn't separate commands, so the extra quotes probably aren't necessary, but I added them to the suggestion just in case there's different behavior in other environments.
```suggestion
        args.push("--pretty=\"format:%cr\t%an\"");
```